templates/web/common/site_layout.tt.html

```diff
@@ -233,17 +233,7 @@
 								<div itemscope itemtype="[% schema_org_itemtype %]">
 									<div class="row">
 										<div class="small-12 column v-space-short">
-											<h1 itemprop="name">[% title %]
-                        <sup>
-                        [% IF request.tagtype == "categories" %]
```

Comment:
All this content is removed because the same content will be available through the facet knowledge panels?

Reply:
yes!